packages/paste-core/components/checkbox/src/Checkbox.tsx

```diff
@@ -190,12 +194,7 @@ const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
             position="relative"
             type="checkbox"
           >
-            <CheckboxIcon
-              element={`${element}_ICON`}
-              checked={mergedChecked}
-              disabled={disabled}
-              indeterminate={indeterminate}
-            />
+            <CheckboxIcon element={`${element}_ICON`} checked={mergedChecked} indeterminate={indeterminate} />
```

Comment:
question(possibly-blocking): this takes out the `disabled={disabled}` prop, was that on purpose?

Reply:
Yea.... the prop wasn't used in the CheckboxIcon component. Was gonna see if anything happened in chromatic to make sure that this is OK.